src/notification/notification.service.ts

```diff
@@ -343,6 +343,10 @@ export class NotificationService {
             .delete();
     }
 
+    async removePostNotifications(post: Post) {
+        await this.db('notifications').where('post_id', post.id).delete();
```

Comment:
Do we need to delete where `in_reply_to_post_id` matches too?

Reply:
Yes 💯 Thanks for pointing this out! Fixed in https://github.com/TryGhost/ActivityPub/pull/672/commits/756195618e85042570bef614556b45ac1a552554